frontend/src/components/Board/Board.tsx

```diff
@@ -1,7 +1,339 @@
+  {
+    id: '2',
+    email: 'mursal@example.com',
+    phone: '123123123',
+    raw_user_meta_data: {
+      username: 'Mursal',
+      given_name: 'Mursal',
+      surname: 'A',
+    },
+  },
+];
+
+interface IColumn {
+  id: string;
+  board_id: string;
+  board_version: number;
+  index: number;
+  name: string;
+  cards: ICard[];
+  created_at: string;
+}
+
+interface ICard {
+  id: string;
+  list_id: string;
+  index: number;
+  user_creator: string;
+  user_assigned: string | undefined;
+  title: string;
+  description: string;
+  deadline: string | undefined;
+  created_at: string;
+}
+
+interface IColumnProp {
+  column: IColumn;
+}
+
+function TaskCard(props: { item: ICard }) {
+  const item = props.item;
+  // console.log(item)
+  return (
+    <>
+      <Card className='m-1' sx={{ maxWidth: 345 }}>
+        <CardContent>
+          <Typography gutterBottom variant='h6' component='div'>
+            {item.title}
+          </Typography>
+          <Typography variant='body2' color='text.secondary'>
+            {item.description}
+          </Typography>
+        </CardContent>
+        <CardActions className='flex flex-col'>
+          <Button className='justify-self-end' size='small'>
+            Edit
+          </Button>
+        </CardActions>
+      </Card>
+    </>
+  );
+}
+
+function TaskColumn(props: IColumnProp) {
+  const column = props.column;
+  const items = props.column.cards
+    .filter((i: ICard) => i.list_id == column.id)
+    .sort((c) => c.index);
+  // console.log("Column: ", {column}, {items})
+  return (
+    <>
+      <div className='flex flex-col h-min-full w-full min-w-[200px] mx-2 px-2 bg-gray-500 rounded-md pt-2'>
+        <div className='w-full bg-gray-100 text-black font-bold text-xl rounded-md text-center border-2 border-gray-700'>
+          {column.name}
+        </div>
+        <div className='w-full h-full overflow-y-auto'>
+          <Droppable droppableId={column.id}>
+            {(provided) => (
+              <div
+                {...provided.droppableProps}
+                ref={provided.innerRef}
+                className='min-h-[95%]'
+              >
+                {items.map((item: ICard, index: number) => (
+                  <Draggable key={item.id} draggableId={item.id} index={index}>
+                    {(provided) => (
+                      <div
+                        ref={provided.innerRef}
+                        {...provided.draggableProps}
+                        {...provided.dragHandleProps}
+                      >
+                        <TaskCard item={item} />
+                      </div>
+                    )}
+                  </Draggable>
+                ))}
+                {provided.placeholder}
+              </div>
+            )}
+          </Droppable>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function TaskBoard() {
+  const [myColumns, setMyColumns] = useState(columns);
+
+  // Handles moving cards
+  function onDragEnd(result: any) {
```

Comment:
Getting a lint warning for the `any` here, any types that you can use from library here?